src/lib/prospectus.ts

```
@@ -0,0 +1,219 @@
+ */
+export function downloadProspectus(metrics: ProspectusMetrics, config: ProspectusConfig): void {
+    const html = generateProspectusHTML(metrics, config);
+    const printWindow = window.open("", "_blank");
+    if (!printWindow) {
+        alert("Please allow pop-ups to download the prospectus.");
+        return;
+    }
+    printWindow.document.open();
+    printWindow.document.write(html);
+    printWindow.document.close();
+    // Wait for images to load before printing
```

ESLint

[error] Delete `··`

(prettier/prettier)